buildSrc/src/main/kotlin/com/datadog/gradle/plugin/gitdiffs/GitDiffsExtension.kt

```diff
@@ -0,0 +1,11 @@
+package com.datadog.gradle.plugin.gitdiffs
+
+import org.gradle.api.Task
+
+class GitDiffsExtension(val deps: MutableMap<Regex, MutableList<Dependency>> = mutableMapOf()) {
```

Comment:
I may have missed something but it seems you never use this class anywhere :/

Reply:
hmmmm I used to use it in the plugin, I will check